### ISSUE TITLE
What to do when release tarball shasums change?

### ISSUE THREAD

**Issue by jasonkarns** [nodejs/Release#396]
I just discovered that the shasums for an old RC have changed since it was originally released.

v6.11.0-rc.1 https://nodejs.org/download/rc/v6.11.0-rc.1/  was originally released in May 2017.

The current files (all but the sunos-x64 tarballs) now have a modified date of Oct 2017 and the shasums don't match what they were originally:

```diff
-aix-ppc64 "https://nodejs.org/download/rc/v6.11.0-rc.1/node-v6.11.0-rc.1-aix-ppc64.tar.gz#f885a3fee36c4f60c94dead92029311bfc203ea03c50de47aa4bf714632f7e93"
+aix-ppc64 "https://nodejs.org/download/rc/v6.11.0-rc.1/node-v6.11.0-rc.1-aix-ppc64.tar.gz#abcf7e6d6c0f666668f70668c9ae9780ed45dedf3408191a9dc56706c1edf0e9"
-darwin-x64 "https://nodejs.org/download/rc/v6.11.0-rc.1/node-v6.11.0-rc.1-darwin-x64.tar.gz#3e7289ddebb90cfd595d9fb9385067dbcd0c5e3e9c9dbafe95611dd5e595f62f"
+darwin-x64 "https://nodejs.org/download/rc/v6.11.0-rc.1/node-v6.11.0-rc.1-darwin-x64.tar.gz#262f588ef8c027ab629f38ff625d3923ccf5932fd9d28d051a0e0d66d6337ffe"
-linux-arm64 "https://nodejs.org/download/rc/v6.11.0-rc.1/node-v6.11.0-rc.1-linux-arm64.tar.gz#0691afbc601ebd8a0a5fd2c41095e2aab29f9a097e7510eba70a2e94ed9b2211"
+linux-arm64 "https://nodejs.org/download/rc/v6.11.0-rc.1/node-v6.11.0-rc.1-linux-arm64.tar.gz#76b5b366393eaec50d40d598c4f7f9848f6a082a59f82c8ff150dd09aab1f743"
-linux-armv6l "https://nodejs.org/download/rc/v6.11.0-rc.1/node-v6.11.0-rc.1-linux-armv6l.tar.gz#f31e5000f1dc2d2b1f24d0b2780775a6201f472730e45f1be9f787a5c0da825c"
+linux-armv6l "https://nodejs.org/download/rc/v6.11.0-rc.1/node-v6.11.0-rc.1-linux-armv6l.tar.gz#a45130a8cfe3302ee635ee6cb82a78ad2387d776a2c25b4cdb2300682ba3f1e2"
-linux-armv7l "https://nodejs.org/download/rc/v6.11.0-rc.1/node-v6.11.0-rc.1-linux-armv7l.tar.gz#00aae11810681f0cdbef29856c13f3f9b1dc58d7e25d19991679dd65e22122b1"
+linux-armv7l "https://nodejs.org/download/rc/v6.11.0-rc.1/node-v6.11.0-rc.1-linux-armv7l.tar.gz#77f48359fceb2d1838d8619d1dbf7088780c5aca8eaa55cf657436c6fb5056bf"
-linux-ppc64le "https://nodejs.org/download/rc/v6.11.0-rc.1/node-v6.11.0-rc.1-linux-ppc64le.tar.gz#3e1d8df484e8f12a1dcdfdf071954f78b5378dc9fc18173fe68d8d9945409ae6"
+linux-ppc64le "https://nodejs.org/download/rc/v6.11.0-rc.1/node-v6.11.0-rc.1-linux-ppc64le.tar.gz#efd58a5c6299e10b6573044ed54d54d8cf5fd83fd63714272d4a980612e214fb"
-linux-ppc64 "https://nodejs.org/download/rc/v6.11.0-rc.1/node-v6.11.0-rc.1-linux-ppc64.tar.gz#2979bd0cc8a08e739be0d179399f3f8fa37408d54dc17fe6e5638df190e5dc1c"
+linux-ppc64 "https://nodejs.org/download/rc/v6.11.0-rc.1/node-v6.11.0-rc.1-linux-ppc64.tar.gz#7e15249ab72b98c33e1bea631bc9ba171e5516dae218d69df45dd9d4a80620c4"
-linux-s390x "https://nodejs.org/download/rc/v6.11.0-rc.1/node-v6.11.0-rc.1-linux-s390x.tar.gz#e46333ff0e3d4cc2a80da97bf731c7585bb58c5c1de0ba25dffab811c9a64d9f"
+linux-s390x "https://nodejs.org/download/rc/v6.11.0-rc.1/node-v6.11.0-rc.1-linux-s390x.tar.gz#d264f84c9a2387a6b6a425d27c044971496598bf40ff429fe8bbe37e15efadbd"
-linux-x64 "https://nodejs.org/download/rc/v6.11.0-rc.1/node-v6.11.0-rc.1-linux-x64.tar.gz#99021e16dcfd3639ce00d056b647b78d8062c1ccd9ce75627a88eb3495d0c6d8"
+linux-x64 "https://nodejs.org/download/rc/v6.11.0-rc.1/node-v6.11.0-rc.1-linux-x64.tar.gz#6ac31783dadc6fa4540d83c5c1eade4b83d252b8e863742d7108bd4e45efd13a"
-linux-x86 "https://nodejs.org/download/rc/v6.11.0-rc.1/node-v6.11.0-rc.1-linux-x86.tar.gz#dd7a8ba748919facade8033e11155ec295584def927a7a2d8785bfc5e2ce26f7"
+linux-x86 "https://nodejs.org/download/rc/v6.11.0-rc.1/node-v6.11.0-rc.1-linux-x86.tar.gz#bd3d38c944838ccdf8d460cf424428b2a0f19620b8270ba7c18df08144e0bf5b"
 sunos-x64 "https://nodejs.org/download/rc/v6.11.0-rc.1/node-v6.11.0-rc.1-sunos-x64.tar.gz#39d80a4fc35ab3c0132fdd5aa58deb8bb2203c561ceb167869da4e98bb9c98a7"
-sunos-x86 "https://nodejs.org/download/rc/v6.11.0-rc.1/node-v6.11.0-rc.1-sunos-x86.tar.gz#980c3182535d82c57c1a034051160acd92bbb13408b9fb91c30b9b881a198087"
+sunos-x86 "https://nodejs.org/download/rc/v6.11.0-rc.1/node-v6.11.0-rc.1-sunos-x86.tar.gz#0e4641017a78fb8fec88ad82b1c06c5a7555a4632902ff3d0d26cb1dea1fb6b4"
 
-"node-v6.11.0-rc.1" "https://nodejs.org/download/rc/v6.11.0-rc.1/node-v6.11.0-rc.1.tar.gz#3f09f6c43975f6cf4a790e0753353fe9055cf56d3e35e7b4c21ebb48fc5c4236"
+"node-v6.11.0-rc.1" "https://nodejs.org/download/rc/v6.11.0-rc.1/node-v6.11.0-rc.1.tar.gz#cd7a62f6c52885bcba0a84613793c030bd00297eeeb16372c6f9db49f5127fbc"
```

I realize it's been a while since those shasums changed (and even longer since the RC was originally released). But can anyone confirm that the current shasums are correct and trustworthy?

And is there a process for publicizing when the tarballs are changed?

**Comment by MylesBorins** [comment]
@jasonkarns we don't offer stability for the tarballs in rc releases as of right now, in fact I think we have even deleted release candidates at times. We do offer stability for all promoted releases

**Comment by jasonkarns** [comment]
That's cool. I wasn't really expecting permanent availability. The main thing I wanted to hopefully confirm is that there wasn't any chance those tarballs were maliciously modified. (realizing that possibility is drastically low considering more high-value releases, if an attacker _did_ compromise the dist/ host)

**Comment by MylesBorins** [comment]
@jasonkarns we have no reason to believe that those tarballs were compromised

I did a bit of digging on the server. v6.12.0-rc.1 was release on the exact same day. My guess, as the releaser of 6.12.0... I perhaps set the version string incorrectly in node_version.h and when I did the RC build it build a new 6.11.0, which in turn over wrote the other release (something that couldn't happen with the production release).

I'm going to go ahead and close this but am definitely interested in talking about it more if you have anything to add. We could potentially increase reliability of our release candidates... but my gut is that we should keep it as is, more flexible
